### DISCUSSION TITLE
WARNING:root:Class 'werkzeug.local.LocalProxy' is not mapped

### DISCUSSION THREAD

**Discussion by qyg20** [apache/superset#26083] [Q&A / Help]
2023-11-23 09:48:25,724:WARNING:root:Class 'werkzeug.local.LocalProxy' is not mapped
172.17.0.1 - - [23/Nov/2023:09:48:25 +0000] "GET /superset/welcome/ HTTP/1.1" 302 201 "http://localhost:8088/login/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/17.1 Safari/605.1.15"
Unable to load SQLAlchemy dialect <class 'pydruid.db.sqlalchemy.DruidHTTPSDialect'>: No module named 'google'

**Comment by musicmuthu** [comment]
I have the same issue in 311 superset.

**Comment by musicmuthu** [comment]
@qyg20  have you resolved this issue ?

**Comment by MohamedRhimii** [comment]
I've got the same issue. Anyone have solved?
App version 3.1.0-dev

**Comment by musicmuthu** [comment]
https://www.restack.io/docs/superset-knowledge-superset-google-module-error

Try this link 
@MohamedRhimii

**Reply by MohamedRhimii** [reply]
This was it. Thanks man, you saved me lots of time!

**Reply by musicmuthu** [reply]
Welcome
